README.md

```diff
@@ -1,18 +1,75 @@
 # SwissTournament
-A Swiss Tournament Pairing Generator written for the [2nd Muenchener Kaffeehaus Chess Tournament](https://www.meetup.com/Chess-Munich/events/rqwkqsyccpbgb/).
+A Swiss Tournament Pairing Generator written for the [3rd Muenchener Kaffeehaus Chess Tournament](https://www.meetup.com/Chess-Munich/events/svkrqsyccqbdb/).
```

Comment:
Can we add both of the links rather than replacing the old one?

Reply:
I was thinking the same, I will update it